.cirrus/static_helm_test.sh

```diff
@@ -0,0 +1,19 @@
+#/bin/bash
+set -euo pipefail
+
+KUBE_VERSION="${KUBE_VERSION:-1.25.0}"
+STATIC_TEST_FOLDER=static_compatibility_test
+
+if ! [ -d "$STATIC_TEST_FOLDER" ]; then
+    echo "$STATIC_TEST_FOLDER folder not found"
+    echo "you are probably not in the root of a chart"
+fi
+
+echo 'Running test for' $KUBE_VERSION
```

Comment:
A more verbose message could help here: "Running static compatibility tests for Kubernetes version $KUBE_VERSION"